### ISSUE TITLE
Strict Tracking protection breaks twitter.com

### ISSUE THREAD

**Issue by garvankeeley** [mozilla-mobile/firefox-ios#5430]
The page will not load at all with strict mode on. The only item being reported as blocked is google-analytics.com (which is not definitive as being the only  blocked item).

```
Content blocker prevented frame displaying https://mobile.twitter.com/home from loading a resource from https://ton.smf1.twitter.com/responsive-web-internal/sourcemaps/web/vendor.7aff2d650bd7961d4.js.map
```

**Comment by garvankeeley** [comment]
This rule is breaking it:
```
{"action":{"type":"block"},"trigger":{"url-filter":"^https?://([^/]+\\.)?twitter\\.com",
"load-type":["third-party"],"unless-domain":
["*crashlytics.com","*digits.com","*fabric.io","*tweetdeck.com","*twitter.jp"]}},
```

**Comment by csuciu** [comment]
Verified as fixed on v19.1(16203)
